### DISCUSSION TITLE
All Feed Items have example.net as URL, Import/Export not possible

### DISCUSSION THREAD

**Discussion by simondueckert** [FreshRSS/FreshRSS#7028] [Q&A]
I have a new phenomenon on my FreshRSS installation (via Yunohost). All feed items have "https://example.net" as URL. I cannot export OPML file (error 500). I have the feat, that the instance has been hacked. Has anyone experienced something similar?

**Comment by Frenzie** [comment]
There are example configs in the docs here along the lines of:
```
<VirtualHost *:80>
	ServerName rss.example.net
	DocumentRoot /path/to/FreshRSS/p/

	<Directory /path/to/FreshRSS/p>
		AllowOverride AuthConfig FileInfo Indexes Limit
		Require all granted
	</Directory>

	ErrorLog ${APACHE_LOG_DIR}/freshrss_error.log
	CustomLog ${APACHE_LOG_DIR}/freshrss_access.log combined

	AllowEncodedSlashes On
</VirtualHost>
```

I figure there was a copy/paste mistake along those lines (or accidentally lack of adjustment).

**Comment by Alkarex** [comment]
@simondueckert Could you post a screenshot of what you see, as well as the Web server logs corresponding to the error 500?
